scipy/interpolate/tests/test_interpolate.py

```diff
@@ -2581,6 +2582,32 @@ def test_fillvalue_type(self):
         RegularGridInterpolator(points, values)
         RegularGridInterpolator(points, values, fill_value=0.)
 
+    def test_broadcastable_input(self):
+        # input data
+        np.random.seed(0)
+        x = np.random.random(10)
+        y = np.random.random(10)
+        z = np.hypot(x, y)
+
+        # x-y grid for interpolation
+        X = np.linspace(min(x), max(x))
+        Y = np.linspace(min(y), max(y))
+        X, Y = np.meshgrid(X, Y)
+        XY = np.vstack((X.ravel(), Y.ravel())).T
+
+        for itype in (NearestNDInterpolator, LinearNDInterpolator,
```

Comment:
nit: could you rename `itype` to `interpolator` or some other more descriptive name? I read the `itype(list(zip` line first and confusingly read it as "integer type".

Reply:
I agree. I fixed it.